### PR TITLE
Add RBAC and make previous material more coherent

### PR DESCRIPTION
**Pull request by mebenn**
If you do merge, maybe you should consider squashing my history as I tend to commit frequently.  I don't think it is necessary to retain all the history.

Only just added a section on RBAC, but there is still lots to do here.

The idea is simple: start off with su, then progress to sudo, then culminate in RBAC.  I tried to be as succinct as possible, but I don't think I've entirely succeeded.

I think the best way to avoid reading a huge admin book, is to mirror the stuff in system-administration in the manual 'Common Tasks' and place the commands in tables with little or no text. I've started doing this.  Anyone agree or disagree?

### CODE REVIEW

File: docs/handbook/systems-administration.md [hunk 1]
```diff
@@ -76,30 +100,123 @@ means to appropriately add a user to use sudo.
...
 
 The user is then prompted for the user's password and a file is checked to
 establish whether the user is permitted to perform the operation.
+The options are explained below.
+
+### Role-Based Access Control (RBAC)
+
+The _all-or-nothing_ power assigned to the root user has its obvious
+limitations.  While sudo is an improvement by limiting root privileges for only
+a single command, a user allowed to use sudo has access to all root commands.
```
Comment (AdrianKoshka) [inline docs/handbook/systems-administration.md]: > a user allowed to use sudo has access to all root commands.

`sudo` can be setup in a very fine grained manner to only permit the execution of certain commands to a user and/or group. Though in most cases, defaults to give a user access to all root commands. Perhaps this is nitpicking personally but maybe:

> a user allowed to use sudo in most cases has access to all root commands.

would be a more appropriate way to word it.

Comment (mebenn) [reply docs/handbook/systems-administration.md]: oh, great volks, thanks for the good feedback. @AdrianKoshka of course you're right, sudo is somewhat more flexible than I've described; but I was concentrating to the grand entrance of RBAC. And, no, you're not nitpicking. However, I would not say that sudo can be _very fine grained_ That's really the problem with sudo, you can associate a couple of commands to a group of users, but it is really not any fun (if i remember correctly). And, of course, you can assign groups to the sudoers list.
Recall, I'm trying to be succinct; don't know how I'll do that when it comes to describing RBAC: authorizations, roles, ...

Comment (makruger) [reply docs/handbook/systems-administration.md]: @mebenn I am happy to commit these changes as they are, and if you care to elaborate further about sudo, that's fine too. As for squashing your commits, I never bothered for the majority of commits I made, and I am not sure how to squash yours (a GIT expert, I am not).

Comment (AdrianKoshka) [reply docs/handbook/systems-administration.md]: @makruger you can choose to squash the commits when you accept the PR.

Comment (makruger) [reply docs/handbook/systems-administration.md]: @AdrianKoshka Thanks, I never realized squashing was an option when merging the PR.

Comment (mebenn) [reply docs/handbook/systems-administration.md]: I'll improve sudo, so lets not commit just yet.

In git there are 2 ways of merging: git merge and git rebase.  Rebase will generally produce a cleaner, nicer history. Its worth looking into if you are integrating branches onto master.  The history will become terribly cluttered if we retain the history  of each and every branch generated via a pull request.

**Pull request merged**